server/api/impl/proxy/expand.go

```diff
@@ -156,7 +159,10 @@ func stripUser(user *model.User, level apps.ExpandLevel) *model.User {
 }
 
 func stripChannel(channel *model.Channel, level apps.ExpandLevel) *model.Channel {
-	if channel == nil || level == apps.ExpandAll {
+	if channel == nil {
```

Comment:
do we need this split?

Reply:
I prefer having it for parallelism with the other functions, and not to make the same mistake in the future if we want to sanitize the channel somehow.